src/test.js

```diff
@@ -334,7 +341,8 @@ Test.prototype = {
 			testId: this.testId,
 
 			// Source of Test
-			source: this.stack
+			// generating stack trace is expensive, so using a getter will help defer this until we need it
+			get source() { return test.stack; }
```

Comment:
I was incorrectly using `this` before. Anyway, should be passing now (and I've probably invalidated your approval with that last push 😬).